### PR TITLE
[mypy] ignore 6

### PR DESCRIPTION
**Pull request by shmuel44**
## Related Issues
Related: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-10508)

## Description
ignore mypy errors in

- AlibabaActionTrailEventCollector
- SiemApiModule
- ReadPDFFileV2
- CyberArkIdentityEventCollector
- DatadogCloudSIEM
- DropboxEventCollector
- Elasticsearch_v2
- FortiSIEM
- FortinetFortiwebVM
- GitHubEventCollector
- GraphQL
- MongoDB
- MongoDBKeyValueStore
- MongoDBKeyValueStore
- MongoDBLog
- Panorama
- PingCastle
- RSANetWitnessv115
- SymantecEDR
- Vertica

### CODE REVIEW

File: Packs/ApiModules/Scripts/SiemApiModule/SiemApiModule.py [hunk 1]
```diff
@@ -163,6 +163,6 @@ def get_last_run(events: list) -> dict:
         """
         return {'after': events[-1]['created']}
 
-    @abstractmethod
+    @abstractmethod  # noqa: B027
```
Comment (dorschw) [inline Packs/ApiModules/Scripts/SiemApiModule/SiemApiModule.py:166]: why is it failing? I vaguely remember us discussing it in the past

File: Packs/FortiSIEM/Integrations/FortiSIEM/FortiSIEM.py [hunk 2]
```diff
@@ -214,8 +214,7 @@ def getEventsByQuery(session, queryData, max_results, extended_data, max_wait_ti
             if len(key["data"]) == 0 or key["data"][0] == "No report results found.":
                 md = "No report results found."
                 break
-            else:
```
Comment (dorschw) [inline Packs/FortiSIEM/Integrations/FortiSIEM/FortiSIEM.py:217]: is this `ruff`'s change, or ours?

**Review by dorschw — APPROVED** [review]

File: Packs/FortiSIEM/Integrations/FortiSIEM/FortiSIEM.py (hunk 2 above)
Comment (shmuel44) [reply Packs/FortiSIEM/Integrations/FortiSIEM/FortiSIEM.py:217]: [ruff](https://docs.astral.sh/ruff/rules/superfluous-else-break/)
But I fixed it manually.

Comment (shmuel44) [reply Packs/FortiSIEM/Integrations/FortiSIEM/FortiSIEM.py:217]: done

**Comment by shmuel44** [comment]
@kobymeir needs force merge becuase DO106 and missing RN

File: Packs/FortiSIEM/Integrations/FortiSIEM/FortiSIEM.yml [hunk 3]
```diff
@@ -290,7 +290,7 @@ script:
       description: ID of the resource group. Run the "fortisiem-get-lists" command to get the ID. command.
     - name: object-info
       required: true
-      description: 'CSV list of key-value pairs of attributes, for example: name=SomeName,lowIp=192.168.1.1,highIp=192.168.1.2'
+      description: 'CSV list of key-value pairs of attributes, for example: name=SomeName,lowIp=192.168.1.1,highIp=192.168.1.2.'
```
Comment (kobymeir) [inline Packs/FortiSIEM/Integrations/FortiSIEM/FortiSIEM.yml:293]:
```suggestion
      description: 'CSV list of key-value pairs of attributes (for example: name=SomeName,lowIp=192.168.1.1,highIp=192.168.1.2).'
```

File: Packs/MongoDB/Integrations/MongoDB/MongoDB.yml [hunk 4]
```diff
@@ -154,7 +154,7 @@ script:
     - description: Name of the collection to query.
       name: collection
       required: true
-    - description: 'A JSON pipeline query to search by in the collection. Pipeline query should by list of dictionaries. For example: [{"key1": "value1"}, {"key2": "value2"}]'
+    - description: 'A JSON pipeline query to search by in the collection. Pipeline query should by list of dictionaries. For example: [{"key1": "value1"}, {"key2": "value2"}].'
```
Comment (kobymeir) [inline Packs/MongoDB/Integrations/MongoDB/MongoDB.yml:157]:
```suggestion
    - description: 'A JSON pipeline query to search by in the collection. Pipeline query should by list of dictionaries. For example: `[{"key1": "value1"}, {"key2": "value2"}]`.'
```

**Review by kobymeir — CHANGES_REQUESTED** [review]

File: Packs/FortiSIEM/Integrations/FortiSIEM/FortiSIEM.yml (hunk 3 above)
Comment (shmuel44) [reply Packs/FortiSIEM/Integrations/FortiSIEM/FortiSIEM.yml:293]: @ShirleyDenkberg, what do you think about this?

File: Packs/MongoDB/Integrations/MongoDB/MongoDB.yml (hunk 4 above)
Comment (shmuel44) [reply Packs/MongoDB/Integrations/MongoDB/MongoDB.yml:157]: @ShirleyDenkberg, what do you think about this?

File: Packs/FortiSIEM/Integrations/FortiSIEM/FortiSIEM.yml (hunk 3 above)
Comment (ShirleyDenkberg) [reply Packs/FortiSIEM/Integrations/FortiSIEM/FortiSIEM.yml:293]: Either way would be OK.

**Comment by ShirleyDenkberg** [comment]
@kobymeir @dorschw @jlevypaloalto Doc review completed.

**Review by kobymeir — APPROVED** [review]

**Pull request merged**